docs/F-Refinements/two-dimensional-arrays.md

```diff
@@ -84,14 +84,14 @@ a[2][0] a[2][1] a[2][2] a[2][3] a[2][4]
 a[3][0] a[3][1] a[3][2] a[3][3] a[3][4] 
 ```
 
-> **Warning**<br/>
+> **Warning**
```

Comment:
Suggestion to add an admonition. For example, you may use the following admonition -> `:::caution`